suzieq/db/parquet/pq_coalesce.py

```diff
@@ -40,7 +41,11 @@ def pq_file_name(self, *args):  # pylint: disable=unused-argument
 
         """
         # Using timestamp rather than date/time string to simplify reads
-        return f'{self.prefix}{self.block_start}-{self.block_end}.parquet'
+        name_suffix = f'{self.block_start}-{self.block_end}.parquet'
+
+        # {i} is replaced by pyarrow with a sequential number. It must be
+        # provided when a name for the output parquet file is provided
+        return f'{self.prefix}' + '{i}-' + name_suffix
```

Comment:
Why not use a single f-string instead of using +?

Reply:
I can't use fstring in this case, as otherwise `i` is interpreted as a variable